aspects/aspects.go

```diff
@@ -310,78 +310,143 @@ func (a *Aspect) Set(databag DataBag, name string, value interface{}) error {
 		Account:    a.bundle.Account,
 		BundleName: a.bundle.Name,
 		Aspect:     a.Name,
-		Field:      name,
+		Field:      request,
 		Cause:      "field not found",
 	}
 }
 
-// Get returns the aspect value identified by the name. If either the named aspect
-// or the corresponding value can't be found, a NotFoundError is returned.
-func (a *Aspect) Get(databag DataBag, name string, value interface{}) error {
-	subkeys := strings.Split(name, ".")
-	for _, accessPatt := range a.accessPatterns {
-		placeholders, ok := accessPatt.match(subkeys)
-		if !ok {
-			continue
-		}
-
-		path, err := accessPatt.getPath(placeholders)
-		if err != nil {
-			return err
-		}
-
-		if !accessPatt.isReadable() {
-			return &InvalidAccessError{RequestedAccess: read, FieldAccess: accessPatt.access, Field: name}
-		}
+// Get returns the aspect value identified by the request. If either the named
+// aspect or the corresponding value can't be found, a NotFoundError is returned.
+func (a *Aspect) Get(databag DataBag, request string, value interface{}) error {
```

Comment:
It guess it would even make sense to just change the type of *interface{}, since values are always returned in a namespace and we never just put some value directly in the interface{} argument